pi5/config.go

```diff
@@ -11,22 +9,10 @@ import (
 
 // A Config describes the configuration of a board and all of its connected parts.
 type Config struct {
-	AnalogReaders     []mcp3008helper.MCP3008AnalogConfig `json:"analogs,omitempty"`
-	DigitalInterrupts []board.DigitalInterruptConfig      `json:"digital_interrupts,omitempty"`
 }
 
 // Validate ensures all parts of the config are valid.
 func (conf *Config) Validate(path string) ([]string, error) {
-	for idx, c := range conf.AnalogReaders {
-		if err := c.Validate(fmt.Sprintf("%s.%s.%d", path, "analogs", idx)); err != nil {
-			return nil, err
-		}
-	}
-	for idx, c := range conf.DigitalInterrupts {
-		if err := c.Validate(fmt.Sprintf("%s.%s.%d", path, "digital_interrupts", idx)); err != nil {
-			return nil, err
-		}
-	}
 	return nil, nil
 }
 
```

Comment:
I can't comment on line 27, but we've still got analog readers in the config. Was that supposed to be removed, too?

Reply:
I don't think so but I wasnt 100% sure about that one. the comment says `LinuxBoardConfig is a struct containing absolutely everything a genericlinux board might need configured.`. So I thought I should keep it there. But if you think it can be removed I can do that

Reply:
It's needed if you're going to have another component in this process that uses it. but because this is a copy of what was in the RDK, intended to be specialized to a single board, it's going to have a bunch of stuff that is no longer used. 

I don't feel strongly: I suspect it's dead code that will make future maintenance harder to reason about, but won't actually hurt anything in the short term. Leave it in for now if it's not clear it's really dead; we can take another look later.